client/src/pages/Feed.js

```diff
@@ -577,6 +586,8 @@ const Feed = (props) => {
     }
   };
 
+  useEffect(() => {}, [ignoreUserLocation]);
```

Comment:
@clubkemp I don't understand this `useEffect()`, is that for reloading or something (pardon my React knowledge here 😀 )

Reply:
I also think we may not need this

Reply:
I think you are both right. I think it was included to re-render if ignoreUserLocation changes can't remember why, but it's not needed. Removed